main/mainworker.cpp

```diff
@@ -12076,7 +12076,8 @@ void MainWorker::HandleLogNotifications()
 	else
 	{
 		itt = _loglines.begin();
-		sTopic = "Domoticz: " + itt->logmessage;
+		sTopic = "Domoticz: Error received";
```

Comment:
Yeah if you only read the topics, then this is not a handy change ;-)

No need to change it of course, but the topic is much to long to read in the inbox, that's why I proposed this change.
And when opening the mail (when topic is fully visible), you see twice the same message (topic and body).

And your change #832 is only sending notifications for errors, so indeed everything is Error received.